JSDemos/Demos/Calendar/Overview/Angular/app/app.component.html

```diff
@@ -79,12 +80,17 @@
       <span>Week number rule</span>
       <dx-select-box
         [dataSource]="weekNumberRules"
+        [inputAttr]="{ 'aria-label': 'Rule' }"
```

Comment:
```suggestion
        [inputAttr]="{ 'aria-label': 'Week Number Rule' }"
```
plz, update in all approaches